src/lib/handlers.js

```diff
@@ -17,6 +24,8 @@ import buildFeedItem from './events/build-feed-item';
 import normaliseEventKeys from './normalise-event-keys';
 import client from '../db/client';
 import shortid from 'shortid';
+import generateFileName from './generate-file-name';
+import extractFileExtension from './extract-file-extension';
 
 export function postEventHandler (req, res, next) { // eslint-disable-line no-unused-vars
```

Comment:
@minaorangina this ignore statement is for next param - we are not using next in this function.